### DISCUSSION TITLE
Setup okay, what now?

### DISCUSSION THREAD

**Discussion by ghost** [neonbjb/tortoise-tts#426] [Q&A]
Sorry to display my ignorance here but now that I have finally installed everything and run setup.py, seems to have worked. Had a battle with dependencies but all are now installed, what do I do next?
How do I use it to do anything, I'm a bit lost as a non programmer who has only gotten into things since gpt4 started helping me?
I want to use it with my gpt3 twitch bot, is that even possible? Idk!
Any and all advice will be welcome!

**Comment by neonbjb** [comment]
Pasting the documentation from apy.py/TextToSpeech or the README.md file into chatgpt and asking it how you could use this API to build your bot is how I would start learning how to do this if I was a non-programmer.
